WallpaperChanger/config.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import os
+import configparser
+import logging
+
+_MAX_CYCLE_SPEED = 3600
+_MIN_CYCLE_SPEED = 5
+class Config(object):
+    """
+    Configuration object for wall paper.
+    """
+    def __init__(self):
+        self._filename = "./wallpaper.conf"
+        self._config = configparser.ConfigParser()
+        self._config['DEFAULT'] = {
+            'cycle-speed': '5',
+            'gallery-directory': os.path.expanduser(r"~\pictures"),
+            'log-directory': os.path.expanduser(r"~\wallpaper.log"),
+            'randomise': 'False'
+        }
```

Comment:
Put into function 
```python
def reset_parameters()
```

Reply:
Reason why is so that it can be called again if the config parameters need to be reset.